package.json

```diff
@@ -154,5 +153,8 @@
     },
     "resolutions": {
         "uglify-es": "3.3.4"
+    },
+    "dependencies": {
+        "sass": "^1.64.1"
```

Comment:
Can we move this to `devDependencies`? Otherwise, it will be included as a transitive dependency by its consumers.

Reply:
Ah my mistake, will do